README.md

```diff
@@ -1 +1,60 @@
-# ballerina-platform-module-ballerina-websub
\ No newline at end of file
+Ballerina WebSub Library
+===================
+
+  [![Build](https://github.com/ballerina-platform/module-ballerina-websub/workflows/Build%20master%20branch/badge.svg)](https://github.com/ballerina-platform/module-ballerina-websub/actions?query=workflow%3A%22Build+master+branch%22)
```

Comment:
Shall we rename this GitHub action as just build? Else the badge will be really long

https://github.com/ballerina-platform/module-ballerina-websub/blob/buddhi-dev/.github/workflows/build-master.yml#L1

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-websub/pull/17/commits/e9fc0613a847130c46dbb769159ab079eab38de2